volatility/framework/symbols/windows/extensions/pool.py

```diff
@@ -16,6 +20,7 @@ class POOL_HEADER(objects.StructType):
     def get_object(self,
                    type_name: str,
                    use_top_down: bool,
+                   kernel_symbol_table: str,
```

Comment:
This is technically a change to the API, so we'd need to make sure that all core plugins which rely on it are changed, and we'd need to bump a version number somewhere, so this bit probably requires a little more consideration.  I'm all for the change I just want to figure out how best to make it...  5:)

Reply:
Ok, so a better way of doing this would be make it optional (as `None`) and then check if it's None and if so, make it symbol_table_name.  Also, I think the variable should be either `symbol_table` (or `symbol_table_name`) to be in keeping with other methods.  If it's symbol_table_name, you'll need to watch out for it being reset by the line above.

After that, we can just make an additive bump to the framework version.  In `constants/__init__.py`, bump the `VERSION_MINOR` variable, and then in any plugin that passes in a value to this method, it will need the class variable set as follows: `_required_framework_verison = (1, <whatever-it-was-bumped-to>, 0)`.

Reply:
Okay, please take a look at https://github.com/volatilityfoundation/volatility3/pull/226/commits/7c584a5955e15a6ce6e5d65bf146b140e72ec717 if this fits. 

Also, I think only `poolscanner` uses this method so I just added the variable to that plugin.

Reply:
Yep, looks good, thanks!  5:)